### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by Nuga0909**
This is the first feature branch of the Leaderboard project. Features:

- Used correct Gitflow
- Used webpack to create files
- Write plain HTML markup with minimum styling (just to make the layout)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <img src="./assets/murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-<h3><b>Microverse Capstone Project - Module 1</b></h3>
+<h3><b>Leaderboard Project</b></h3>
```
Comment (DammyShittu) [inline README.md:9]: - [ ]  In the Getting Started section of the readme doc kindly fix your link here:

  image

File: src/index.html [hunk 2]
```diff
@@ -1,11 +1,49 @@
...
+      </div>
+      <div class="item form-container">
+        <form action="">
+          <h2>Add your score</h2>
+          <input type="text" id="name" name="name" placeholder="Your name" required/>
+          <input type="number" id="number" name="number" placeholder="Your score" required/>
+          <div class="btn btn-submit">
+            <button type="submit">Submit</button>
+          </div>
+        </form>
```
Comment (DammyShittu) [inline src/index.html]: - [ ] Kindly ensure your app layout matches the wireframe.

  Your layout | Wireframe layout
  :---:|:---:
  ddud | yydd

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Nuga0909,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Nuga0909  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in making the required changes :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**